pom.xml

```diff
@@ -325,6 +325,24 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.mortbay.jetty</groupId>
```

Comment:
Removing the exclusion is better, as that way we'll be assured to get the correct version, and it will be automatically removed if a new version of Hadoop stops using it.